Sources/TraceLog/LogLevel.swift

```diff
@@ -0,0 +1,89 @@
+///
+///  LogLevel.swift
+///
+///  Copyright 2016 Tony Stone
+///
+///  Licensed under the Apache License, Version 2.0 (the "License");
+///  you may not use this file except in compliance with the License.
+///  You may obtain a copy of the License at
+///
+///  http://www.apache.org/licenses/LICENSE-2.0
+///
+///  Unless required by applicable law or agreed to in writing, software
+///  distributed under the License is distributed on an "AS IS" BASIS,
+///  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+///  See the License for the specific language governing permissions and
+///  limitations under the License.
+///
+///  Created by Tony Stone on 4/23/16.
+///
+import Swift
+import Foundation
+
+///
+/// LogLevels
+///
+/// Note: this enum must support both Swift and ObjC
+///
+@objc public enum LogLevel : Int, Comparable {
+    case off     = 0
```

Comment:
Does a LogLevel `off` turn off all logs for that module? If so, we should try to discourage that usage. It would be nice if we can say:
* If logging is ON, then *error* logs will definitely show up.
* If you don't want error logs, then turn off logging completely.